code/game/turfs/simulated/floor/reinf_floor.dm

```diff
@@ -33,6 +32,9 @@
 /turf/open/floor/engine/try_replace_tile(obj/item/stack/tile/T, mob/user, params)
 	return
 
+/turf/open/floor/engine/remove_tile(mob/user, silent = FALSE, make_tile = TRUE)
+	return
```

Comment:
```suggestion
	.=..()
	ScrapeAway(flags = CHANGETURF_INHERIT_AIR)
```

Reply:
No that makes it be stupid and counter the point of reinforced floors being nearly indestructible by any means